buildSrc/src/main/kotlin/io/spine/internal/gradle/Runtime.kt

```diff
@@ -26,17 +26,17 @@
 
 package io.spine.internal.gradle
 
-import com.google.common.base.Joiner
-import io.spine.internal.dependency.Flogger
 import java.io.File
 import java.io.InputStream
 import java.io.StringWriter
+import java.lang.ProcessBuilder.Redirect.PIPE
 import java.util.*
 
-object Runtime {
-    @Suppress("unused")
-    val flogger = Flogger.Runtime
-}
+/**
+ * Utilities for working with processes from Gradle code.
+ */
+@Suppress("unused")
+private const val ABOUT = ""
```

Comment:
Shouldn't we say something here?

Reply:
The point of having this constant is to have a KDoc, which covers the purpose of the file.  We do it this way in some other files under `buildSrc`.

Having the constant, in turn, also allows to avoid renaming the file. Now, after removing the `Runtime` object from this file, there's only one top-level declaration which is `Cli` class. detekt insists on having the file named after single top-level declaration. That's why I added the constant.